### ISSUE TITLE
[RN 0.46.0 Compatibility] Unable to resolve module ReactNative

### ISSUE THREAD

**Issue by joncursi** [inProgress-team/react-native-meteor#248]
Hitting this packager issue after upgrading to RN 0.46.0:

![screen shot 2017-07-06 at 8 20 47 pm](https://user-images.githubusercontent.com/4974609/27937921-a54bb5d0-6288-11e7-8d98-d31d8044208c.png)

**Comment by joncursi** [comment]
It looks like these files have moved into a `shims` folder:

![screen shot 2017-07-06 at 8 23 24 pm](https://user-images.githubusercontent.com/4974609/27937988-07091c9a-6289-11e7-9002-54b40e458897.png)

**Comment by joncursi** [comment]
Changing this line:

https://github.com/inProgress-team/react-native-meteor/blob/master/src/Data.js#L1

to

```
import ReactNative from 'react-native/Libraries/Renderer/shims/ReactNative';
```

temporarily fixes the issue, however this is probably not the right way to do it in RN `0.46.0` since we should be using the Fiber renderer

**Comment by Nauzer** [comment]
Will be fixed with https://github.com/inProgress-team/react-native-meteor/pull/247

**Comment by Nauzer** [comment]
Solved with latest 1.1.0 Release
